Makefile

```diff
@@ -30,9 +28,11 @@ test-unit: clean
 .PHONY: package
 package:
 	@test -s ./$(artifact_name).jar || { echo "ERROR: Service JAR not found"; exit 1; }
+	$(info Packaging version: $(version))
+	mvn versions:set -DnewVersion=$(version) -DgenerateBackupPoms=false
+	mvn package -DskipTests=true
 	$(eval tmpdir:=$(shell mktemp -d build-XXXXXXXXXX))
 	cp ./start.sh $(tmpdir)
-	cp ./routes.yaml $(tmpdir)
```

Comment:
We'll still need the routes.yaml in the packaged zip, won't we?

Reply:
you are right